### PR TITLE
Tidy Up

### PR DESCRIPTION
**Pull request by joshuatcasey**
See commit history for details

### CODE REVIEW

File: go.mod
Comment (robdimsdale) [inline go.mod:17]: @joshuatcasey I'm surprised we don't have to pin this to `v0.6.0` like we did everywhere else. I accept that it seems to work, but i was wondering if you had thoughts on this?

Comment (joshuatcasey) [reply go.mod:17]: There's a `replace` directive at the bottom of `go.mod`. Rather confusingly, `go mod` updates the `require` directive with the latest version, but the `go.sum` seems to keep the `replaced` version.

Comment (robdimsdale) [reply go.mod:17]: Ah, yes, i missed because it wasn't shown in the diff by default (it didn't change) and it was at the bottom whereas in other `go.mod` files we've explicitly put it at the top.

I'll approve this PR.

**Review by robdimsdale — APPROVED** [review]

**Comment by robdimsdale** [comment]
@joshuatcasey I'll leave it up to you as to whether you rebase or squash when you merge.

**Comment by joshuatcasey** [comment]
@robdimsdale I can't merge this PR, could you do that?

I'm ok with squashing since the squashed commit has both the information and the link back to this PR.

**Comment by robdimsdale** [comment]
Sorry @joshuatcasey - I thought you were in @paketo-buildpacks/ruby-contributors . I'll squash and merge now.

**Pull request merged**
